Source/Resources/Content/AzureManagement/MicrosoftResources/ResourceGroups/m365alertsactiongroup.json

```diff
@@ -1,6 +1,7 @@
 {
   "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
   "contentVersion": "1.0.0.0",
+  "description":"What is it: Creates an Azure Monitor Action group to notify specified users of an alert. \n Simeon Notes: Keep your tenant secure by being notified immediately of suspicious activity. Simeon's baseline includes alerting the email address alerts@tenantname.com. \n Learn More: https://docs.microsoft.com/en-us/azure/azure-monitor/platform/action-groups",
```

Comment:
**What does this do?**
Creates an Azure Monitor Action group to notify specified users of an alert. The baseline includes alerting the email address 'alerts@tenantname.com'. 

**Why should you use this?**
This keeps your tenant secure by immediately notifying you of suspicious activity.

**What is the end-user impact?**
N/A

**Learn more**
[Create and manage action groups in the Azure portal](https://docs.microsoft.com/en-us/azure/azure-monitor/platform/action-groups) @metamenos should this link be different or more specific? it's the same as the previous one

Reply:
I like the link. It defines what an action group is and if they read on also tells how they can use the action group.